### PR TITLE
Replace podcast icon with Amazon logo

### PR DESCRIPTION
**Pull request by WebSnke**
As mentioned in #108.

### CODE REVIEW

**Comment by WebSnke** [comment]
@juliaferraioli @amcasari @perriefidelis 
Could one of you take a look at my pull request? Thanks.

File: layouts/partials/svg/amazon.svg
Comment (juliaferraioli) [inline layouts/partials/svg/amazon.svg:1]: Looking great! Can you please update the SVG to use this in the opening SVG tag? That lets it scale properly.

`<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512" {{ with .size }} height="{{ . }}" {{ end }} >`

**Review by juliaferraioli — CHANGES_REQUESTED** [review]
Thanks for pinging the issue -- I didn't get notified of your PR for some reason. I'll look into that.

File: layouts/partials/svg/amazon.svg
Comment (WebSnke) [reply layouts/partials/svg/amazon.svg:1]: I just did and the SVG does not show up properly anymore...

Comment (juliaferraioli) [reply layouts/partials/svg/amazon.svg:1]: But it works great! It's the magic of hugo templates. I probably should change it eventually but it's not a high priority right now.

**Review by juliaferraioli — APPROVED** [review]
Looks great! If you take the pr out of draft mode I can merge it.

@all-contributors, please add @WebSnke for coding contributions!

File: layouts/partials/svg/amazon.svg
Comment (juliaferraioli) [reply layouts/partials/svg/amazon.svg:1]: Oh, I triggered the deploy preview, so you can see it rendering: https://deploy-preview-117--opensourcestories.netlify.app/

**Comment by juliaferraioli** [comment]
Let's try this again...

@all-contributors, please add @WebSnke for coding contributions!

**Pull request merged**